tests/integration/rdb.tcl

```diff
@@ -30,7 +30,7 @@ start_server [list overrides [list "dir" $server_path "dbfilename" "encodings.rd
 "0","set_zipped_2","set","100000","200000","300000","400000",
 "0","set_zipped_3","set","1000000000","2000000000","3000000000","4000000000","5000000000","6000000000",
 "0","string","string","Hello World"
-"0","zset","zset","a","1","b","2","c","3","aa","10","bb","20","cc","30","aaa","100","bbb","200","ccc","300","aaaa","1000","cccc","123456789","bbbb","5e+9",
```

Comment:
this revert the side effect change in 29380ff77d

Reply:
yep, but it's because it can be represented as an integer, which is a lot faster to compute.